tests/test_env_parsing.py

```diff
@@ -0,0 +1,15 @@
+from classy_config import ConfigValue
+from classy_config.config import get_raw_config, register_config
+
+test_values = [
+    "1",
+    "2",
+    "3",
+    "4"
+]
+
+def test_env_config():
+    register_config(filepath="tests/configs/test-config.env", prefix="env")
+
+    for count, test_val in enumerate(test_values):
+        assert ConfigValue(f"env.test_{count + 1}", str) == test_val
```

Comment:
`enumerate` has `start=` that could be utilised to save having to add 1
https://docs.python.org/3/library/functions.html#enumerate

```suggestion
    for count, test_val in enumerate(test_values, start=1):
        assert ConfigValue(f"env.test_{count}", str) == test_val
```